node/network/statement-distribution/src/lib.rs

```diff
@@ -1347,11 +1347,20 @@ async fn handle_incoming_message_and_circulate<'a>(
 		// were not aware of before, we cannot have any dependent statements from the candidate.
 		let _ = metrics.time_network_bridge_update_v1("circulate_statement");
 
-		let session_index = runtime
-			.get_session_index_for_child(ctx.sender(), relay_parent)
-			.await
-			.unwrap_or(0_u32);
-		let topology = topology_storage.get_topology_or_fallback(session_index);
+		let session_index = runtime.get_session_index_for_child(ctx.sender(), relay_parent).await;
+		let topology = match session_index {
+			Ok(session_index) => topology_storage.get_topology_or_fallback(session_index),
+			Err(e) => {
+				gum::error!(
```

Comment:
We're logging runtime errors as debug elsewhere. They can happen due to a block being pruned e.g. on finalization.